README.md

```diff
@@ -1,27 +1,50 @@
-Ubuntu documentation service
-===
+# Ubuntu documentation service
 
 [![Build Status](https://travis-ci.org/ubuntudesign/docs.ubuntu.com.svg?branch=master)](https://travis-ci.org/ubuntudesign/docs.ubuntu.com)
 
 A central platform for hosting Ubuntu's various documentation portals.
 
-Basic usage
----
+## Local development
+
+### Dependencies
+
+1. [Install Docker](https://www.docker.com/community-edition#/download)
+2. Get [the `repo` command](https://github.com/esrlabs/git-repo), e.g.:
```

Comment:
This whole section can be replaced by `snap install git-repo` (upon install, `git-repo.repo` is automatically aliased to the `repo` command).